Packs/LogPoint_SIEM_Integration/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### LogPoint SIEM Integration
+
+- %%UPDATE_RN%%
```

Comment:
Don't forget to update the release notes of the changes that you did.

Reply:
I thought I'd already added it, maybe I missed something?

Reply:
You can send me the line and I'll add it for you

Reply:
**Fixing log search**

- Update search_logs_command function
- Update get_searchid_command function
- Add timeout argument to the lp-get-searchid command

Reply:
Add the release notes according to the following documentation https://xsoar.pan.dev/docs/documentation/release-notes#general.

Reply:
I tried to update the release note but without success, please update it yourself as you suggested earlier.